torch/backends/cudnn/rnn.py

```diff
@@ -180,8 +180,10 @@ def get_parameters(fn, handle, weight_buf):
 def _copyParams(params_from, params_to):
     for layer_params_from, layer_params_to in zip(params_from, params_to):
         for param_from, param_to in zip(layer_params_from, layer_params_to):
-            assert param_from.type() == param_to.type()
-            param_to.copy_(param_from)
+            assert not ((param_from is None or param_from.dim() == 0) ^ (param_to is None or param_to.dim() == 0))
+            if not ((param_from is None or param_from.dim() == 0) and (param_to is None or param_to.dim() == 0)):
```

Comment:
```python
has_from = param_from is None or param_from.dim() == 0
has_to = ...
assert not (has_from ^ has_to)
if not has_from:  # NOTE: has_from == has_to because of the assert
    continue
assert param_from.type() == param_to.type()
param_to.copy_(param_from)
```